autogpt_platform/backend/.env.example

```diff
@@ -15,6 +15,9 @@ REDIS_PORT=6379
 REDIS_PASSWORD=password
 
 ENABLE_CREDIT=false
+STRIPE_API_KEY=
+STRIPE_WEBHOOK_SECRET=
+
```

Comment:
Should we set `PLATFORM_BASE_URL` https://github.com/Significant-Gravitas/AutoGPT/pull/9050/files#diff-62020caf1b9a15e0e3b9b3b1b69d5f6464bf7643f62354cbbaabf755d57b6064L39 using local default value ? e.g: `http://localhost:3000` ?